playerbot/strategy/triggers/GenericTriggers.h

```diff
@@ -272,6 +272,13 @@ namespace ai
         virtual Value<Unit*>* GetTargetValue();
     };
 
+    class BuffCanBeCastTrigger : public BuffTrigger
+    {
+    public:
+        BuffCanBeCastTrigger(PlayerbotAI* ai, string spell, int checkInterval = 1, bool checkIsOwner = false)
+            : BuffTrigger(ai, spell, checkInterval, checkIsOwner) {}
+        virtual bool IsActive();
+    };

```

Comment:
needed? Does the same as SpellCanBeCastTrigger and is never used

Reply:
it is needed and im using it in the "LowOrCriticalHealthAndAndNaturesSwiftnessTrigger" trigger of the druid

The spellcanbecast trigger only allows to be casted on targets, which are not "yourself". Buff trigger is overloading to "look at your char"